helm-charts/private-nginx_values.yaml

```diff
@@ -0,0 +1,406 @@
+## nginx configuration
```

Comment:
maybe we can have here only the custom values the other we can remove

Reply:
Left only custom values for both nginx and prometheus.